src/main/flight/pid_init.c

```diff
@@ -423,6 +425,9 @@ void pidInitConfig(const pidProfile_t *pidProfile)
 #endif
 
     pidRuntime.levelRaceMode = pidProfile->level_race_mode;
+    pidRuntime.tpaBreakpoint = (pidProfile->tpa_breakpoint - PWM_RANGE_MIN) * 0.001f; // .35 from 1350, range can be 0 to 0.99, limited by CLI range
+    pidRuntime.tpaBreakpoint = constrainf(pidRuntime.tpaBreakpoint, 0.0f, 0.99f); // limit range to 0 to 0.99
```

Comment:
Not important here, but single  pidRuntime.tpaBreakpoint write is slightly cleaner ...

Reply:
Done, is better